packages/components/ng-ovh-contacts/LICENSE

```diff
@@ -1,29 +0,0 @@
-BSD 3-Clause License
```

Comment:
This change doesn't look related to the PR's purpose

Reply:
Thanks for pointing out this. It has been resolved via the rebase.